sefaria/model/text.py

```diff
@@ -1334,12 +1334,12 @@ class Version(AbstractTextRecord, abst.AbstractMongoRecord, AbstractSchemaConten
         "purchaseInformationImage",
         "purchaseInformationURL",
         "hasManuallyWrappedRefs",  # true for texts where refs were manually wrapped in a-tags. no need to run linker at run-time.
-        "actualLanguage",
-        'languageFamilyName',
-        "isBaseText",
-        'isSource',
-        'isPrimary',
-        'direction', # 'rtl' or 'ltr'
+        "actualLanguage",  # ISO language code
+        'languageFamilyName',  # full name of the language, but without specificity (for Judeo Arabic actualLanguage=jrb, languageFamilyName=arabic
+        "isBaseText",  # should be deprecated (needs some changes on client side)
```

Comment:
can probs remove this line now

Reply:
no. we should deploy, then remove it from data, and then we can remove this line (and say everyone to remove it from data, doing it on shmuels' cauldron, etc.)

Reply:
Sure